cloud/cloud.go

```diff
@@ -369,14 +357,14 @@ func Put[T Resource](ctx context.Context, client *Client, payload interface{}, u
 
 // Request makes a request to the Terramate Cloud using client.
 // The instantiated type gets decoded and return as the entity T,
-func Request[T Resource](ctx context.Context, c *Client, method string, url url.URL, postBody io.Reader) (entity T, err error) {
+func Request[T Resource](ctx context.Context, c *Client, method string, url url.URL, payload any) (empty T, err error) {
```

Comment:
This returned variable is misleading, could we rename it? From my perspective, it sounds like always empty response.

Reply:
```suggestion
func Request[T Resource](ctx context.Context, c *Client, method string, url url.URL, payload any) (res T, err error) {
```

Reply:
btw, it's not blocking to approve the PR

Reply:
all good, I improved docs as well.
thanks